index.js

```diff
@@ -27,11 +27,16 @@ module.exports = function(content) {
     return callback(null, content);
   } else {
     var plugins = [];
-    plugins.push(imageminGifsicle(options.gifsicle));
-    plugins.push(imageminMozjpeg(options.mozjpeg));
-    plugins.push(imageminSvgo(options.svgo));
-    plugins.push(imageminPngquant(options.pngquant));
-    plugins.push(imageminOptipng(options.optipng));
+    if(options.gifsicle.enabled !== false)
```

Comment:
What's the reason for using false here?
`options.gifsicle = config.gifsicle || false,`

We could use {} instead:
`options.gifsicle = config.gifsicle || {},`